MSSQL/OMOPLoader.sql

```diff
@@ -1718,13 +1768,13 @@ and M.provider_id=l.provider_id
 and M.concept_cd=l.concept_Cd
 and M.start_date=l.start_Date
  
-WHERE  m.MODIFIER_CD='@'
+WHERE  m.MODIFIER_CD='@';
+
+SET @END_TIME = GETDATE();
+PRINT 'OMOPMEASUREMENT_LAB EXECUTION TIME: ' +  CAST(datediff(s, @start_time, @end_time) as nvarchar(50));
 
-;drop table temp_mod_priority; 
-drop table temp_mod_result_loc;
 
 END
-GO  

```

Comment:
Again, why get rid of the go?

Reply:
Again a typo, reverted in the latest commit.